lib/run_loop/core.rb

```diff
@@ -68,7 +68,8 @@ def self.run_with_options(options)
       instruments = options[:instruments] || RunLoop::Instruments.new
 
       # Find the Device under test, the App under test, UIA strategy, and reset options
-      device = RunLoop::Device.detect_device(options, xcode, simctl, instruments)
+      # Experimental selected device detection
+      device = RunLoop::DetectAUT::Xcode.detect_selected_device || RunLoop::Device.detect_device(options, xcode, simctl, instruments)
```

Comment:
Yes I know but i have in mind like use some switch like `TRY_TO_FIND=1` and only if it set to true try to find selected device, if not detect in old way.
And i don't want to mess in `detect_device` method now.